draft-irtf-cfrg-hpke.md

```diff
@@ -270,36 +270,41 @@ following operations:
 - GenerateKeyPair(): Generate an ephemeral key pair `(sk, pk)`
   for the DH group in use
 - DH(sk, pk): Perform a non-interactive DH exchange using the
-  private key sk and public key pk to produce a shared secret
-  of length Npk
+  private key sk and public key pk to produce a Diffie-Hellman
+  shared secret of length Npk
 - Marshal(pk): Produce a fixed-length byte string
   encoding the public key `pk`
 - Unmarshal(enc): Parse a fixed-length byte string to recover a
   public key
 
+And consider the following additional definition:
+
+- Hash'(m): Compute the cryptographic hash of input message `m`
+  producing the shared secret of length Nzz returned by the algorithm
```

Comment:
`Nzz` needs to be defined somewhere; TBH seems like this would be good to know for other KEMs as well.  I think this loses the idea that the DH output is fixed-size, though.  Do we need that?

Reply:
Putting it alongside the `Nh` et al. definitions would suffice. 

> I think this loses the idea that the DH output is fixed-size, though. Do we need that?

@bifurcation I'm not sure I follow. Nzz *is* fixed, so isn't the output fixed size?

Reply:
Good point, I'll add `Nzz` to the “Cryptographic Dependencies” part; I am not sure what you mean regarding the fixed-size DH output?

Reply:
Might be relevant: Encap and AuthEncap now have the same output size for `zz`. Before, AuthEncap's `zz` had double the size of Encap's `zz`.

Reply:
Interesting -- does that affect analysis?

Reply:
Not so much I'd say – it makes analysis of Extract a bit easier because it's input length has one case distinction less.